EasyBusiness/package.json

```diff
@@ -6,8 +6,10 @@
     "antd": "^3.23.2",
     "babel-plugin-import": "^1.12.1",
     "customize-cra": "^0.7.0",
+    "env2": "^2.2.2",
```

Comment:
The two packages you've installed don't have anything to do with the design of the login form, try to make commits that only relate to a single task.